index.es.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
```

Comment:
I would rather not maintain a `index.es.js` and a `index.js`, can we reduce it to just one file, by changing the export in `index.js` to

```js
module.exports = module.exports.default = isMobile
```

Reply:
Actually that would be even better =)
I'll edit the commit